src/ctia/stores/es/verdict.clj

```diff
@@ -0,0 +1,8 @@
+(ns ctia.stores.es.verdict
+  (:require [ctia.stores.es.crud :as crud]
+            [ctim.schemas.verdict :refer [StoredVerdict]]))
+
+(def handle-create-verdict (crud/handle-create :verdict StoredVerdict))
+(def handle-read-verdict (crud/handle-read :verdict StoredVerdict))
+(def handle-delete-verdict (crud/handle-delete :verdict StoredVerdict))
+(def handle-list-verdicts (crud/handle-find :verdict StoredVerdict))
```

Comment:
Yes that was exactly my thought, it's really impressive to witness how much you can achieve now on Ctia with so little code.